app/assets/stylesheets/base/base/_forms.scss

```diff
@@ -0,0 +1,78 @@
+fieldset {
+  background: lighten($base-border-color, 10);
+  border: $base-border;
+  margin: 0 0 ($base-spacing / 2) 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 $base-spacing / 2` instead of `0 0 $base-spacing / 2 0`